workshops/book-dash/meeting-notes/2023-09-05-WG.md

```diff
@@ -0,0 +1,57 @@
+* (Arielle) Upload notes to GitHub repo
+* (Arielle) communicate recommendations to core delivery team 
+
+## Notes
+* Book Dash working group vs planning committee team
+    * Assuming that we would still have RPM and RCM on planning committee, do need insight into operations and what is possible 
+    * (Recommendation) Planning committee - invite previous attendees to apply for next round as part of the follow up to next time 
+    * Working group - how are people selected/or how to do they apply? 
+        * Recommendations for basic criteria
+            * Have attended a Book Dash 
+            * [desirable] Have previously been on the planning committee at least once 
+        * Term length for working group: Two years? Would balance effort on elections/selections, with keeping the WG fresh 
+        * Could revisit options around renewing terms later 
+        * How to engage new people in between rounds? Planning committee/hubs? 
+        * How to manage people who **only** engage as part of the Book Dash? (Q for others as well)
+* Review applications to become part of the planning committee. Arielle has the access
+    * EK & AB would recommend accepting all to join the planning committee for this round - with the expectation that after the Book Dash some will join the Working Group
+    * Logic of this is due to timings an d lack of governanced docs difficult to stand up the WG right now 
+* Start brainstorming any activities that we would like to offer / to start quoting for the Book Dash this round
+    * Discussion around Scriberia images are sometimes duplicated and we struggle to get them into the turing Way itself - should we include them this time or skip and focus on imagathon?
+    * EK: maybe we alternate Book Dashes for Scriberia and go back to a longer time period for the May round
+    * Also recommend we have "imagathons" during the Book Dash this time to help get images into the guides - good for newer team members 
+    * JonnyWritesWords - recommend two sessions due to timezone differences
+    * Social session ideas: learning british sign language, data visualisation, design and accessibility (could we find professional instructors for these?) 
+    * Session in Spanish or different language rooms - possibly with live translation to English?
+      *  Need to speak to localisation team about recommendations for translators and whether Spanish is a good language to start with 
+* Identify or start the conversation about the max number of participants per hub and nr of hubs / rules to open up a hub
+    * Esther not able to join this meeting , would appreciate her input 
+    * Draft Recommendations 
+        * Hub operate on 1-2 days max (have those days coordinated across the Book Dash, to balance online and in-person collaborations) 
+        * In person limits - max 15, recommend 10 to help with coordination, and stress levels!
```

Comment:
I agree with the max 1-2 days for a local hub. I think participation could be up to 20 participants - we managed with about 10-15 participants for the TU Delft hub and I think the Turing hubs were around 15-20 participants before

Reply:
Agree with max 10 in-person.

Reply:
Just to clarify, max 20 per hub? But max 10 in person?

Reply:
Max 10 in-person. Together, virtual and in-person, max 20